docs/playbook/data-quality-testing-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Guide to testing data quality in Glue Jobs
+description: "A guide to continuous data quality testing in Glue Jobs"
+layout: playbook_js
+tags: playbook
+---
+
+## Resources
+- [PyDeequ README][pydeequ-readme]
+
+## How?
+
+- Need to include `s3://dataplatform-stg-glue-scripts/jars/deequ-1.0.3.jar` in the glue job extra jars (--extra-jars).
+- Need to include `s3://dataplatform-stg-glue-scripts/python-modules/pydeequ-1.0.1.zip` in the glue job extra python files (--extra-py-files).
+- Write your repository data to the S3 location which uses the template format `s3://dataplatform-stg-EXAMPLE-zone/quality-metrics/department=EXAMPLE/dataset=EXAMPLE/deequ-metrics.json`
+
+### Example Check
+
+Here is an example of using deequ checks to validate a dataframe, and storing related metrics to S3.
+The `description_of_work` column is checked to be complete, and `work_priority_priority_code` between
+1 and 4 inclusively.
+
+```python
+metrics_target_location = "s3://dataplatform-stg-refined-zone/quality-metrics/department=housing-repairs/dataset=tv-aerials-cleaned/deequ-metrics.json"
```

Comment:
We can maybe set a Glue Job parameter for the metrics target location and in the Prerequisites section above we can instruct them to update it